pkgs/applications/audio/qmidiarp/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchgit
+, automake
+, autoreconfHook
+, lv2
+, pkg-config
+, qt5
```

Comment:
You should not take qt5 in argument, use libsForQt5.callPackage instead: see https://nixos.org/nixpkgs/manual/#sec-language-qt

Reply:
I'm a little afraid of this becoming circular as I'm following advise from comments from an earlier PR.

Reply:
Is there a reason that one way would be preferable to the other?

Reply:
I am not sure about the specifics, but this has something to do with preventing you from pulling dependencies built with different versions of qt by mistake.
We can wait for the review of someone more knowledgeable than me in qt things.

Reply:
Sure. Thanks very much for looking at this!

Reply:
I prefer just using the default `mkDerivation` and `callPackage`. As long as you access all Qt dependencies through the same argument, decoherence should not occur. Using Qt’s own functions is not really composable, see https://github.com/NixOS/nixpkgs/pull/71089.

Reply:
Thanks